exonum-java-binding/fakes/src/main/java/com/exonum/binding/fakes/services/service/TestService.java

```diff
@@ -62,12 +70,33 @@ protected TestSchema createDataSchema(View view) {
   @Override
   public void initialize(Fork fork, Configuration configuration) {
     TestSchema schema = createDataSchema(fork);
-    ProofMapIndexProxy<HashCode, String> testMap = schema.testMap();
+    ProofMapIndexProxy<HashCode, String> testMap = schema.initializeServiceMap();
     testMap.put(INITIAL_ENTRY_KEY, INITIAL_ENTRY_VALUE);
   }
 
   @Override
   public void createPublicApiHandlers(Node node, Router router) {
     // No-op: no handlers.
   }
+
+  @Override
+  public void beforeCommit(Fork fork) {
+    try {
+      TestSchema schema = createDataSchema(fork);
+      ProofMapIndexProxy<HashCode, String> map = schema.beforeCommitMap();
+      String value = map.get(BEFORE_COMMIT_ENTRY_KEY);
+      Integer newValue = (value != null)
+          ? Integer.valueOf(value) + 1
+          : BEFORE_COMMIT_INITIAL_VALUE;
+      map.put(BEFORE_COMMIT_ENTRY_KEY, newValue.toString());
+    } catch (Throwable th) {
```

Comment:
Unneeded: com.exonum.binding.core.runtime.ServiceRuntime.beforeCommit already logs the exceptions
          in before commit.